src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [status-im2.common.plus-button.view :as plus-button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]))
+
+(defn user-account-view
+  [{:keys [name balance percentage-value amount customization-color watch-only type]}]
+  [rn/view
+   [rn/view (style/card customization-color watch-only)
+    [rn/view style/profile-container
+     [icon/icon :contact {:size 20}]
```

Comment:
:size 20 is default, so its redundant here

Reply:
tks, its removed